generators/client/templates/react/src/main/webapp/app/config/notification-middleware.ts.ejs

```diff
@@ -101,6 +101,9 @@ export default () => next => action => {
                 const fieldErrors = data.fieldErrors;
                 for (i = 0; i < fieldErrors.length; i++) {
                   const fieldError = fieldErrors[i];
+                  if (['Min', 'Max', 'DecimalMin', 'DecimalMax'].includes(fieldError.message)) {
```

Comment:
would it be possible to add unit test for react as well?

Reply:
Done